content/applications/sales/sales/gelato.rst

```diff
@@ -4,4 +4,166 @@
+- Configure delivery options in Odoo and receive order updates via webhooks.
+
+Configuration
+=============
+
+.. important::
+   The company information (Company Name and Billing address) in the Gelato account *must* match the
+   company information in the Odoo database in order for sales orders to be confirmed and sent to
+   Gelato for fulfillment.
+
+   .. image:: gelato/gelato-company.png
+      :alt: Company information in Gelato.
+
+   .. image:: gelato/odoo-company.png
+      :alt: Company information in Odoo.
+
+Configure API Keys and Webhooks in Gelato
+-----------------------------------------
+
+Before configuring the Gelato connector in Odoo, first obtain API credentials and webhooks from the
+Gelato account.
+
+API connectors enable Odoo **Sales** to send and receive data from Gelato for order processing,
+while webhooks provide real-time updates on order status and shipment tracking.
+
+API Key
+~~~~~~~
+
+An API Key is a unique authentication token that allows Odoo to securely communicate with Gelato’s
+API, enabling order transmission, status updates, and data synchronization.
+
+After logging into Gelato, click :icon:`fa-code` :guilabel:`Developer` in the left menu bar. From
+here, click on :guilabel:`API keys`. In the new page, click the :guilabel:`Add API Key` button to
+open a new API key form. Type in a name, then click :guilabel:`Create Key`.
+
+Copy the generated API key using :guilabel:`Copy to Clipboard`.
+
+.. image:: gelato/gelato-api-key.png
+   :alt: Newly generated API key in the Gelato platform.
+
+.. important::
+   Copy the API key and store on a notepad before leaving this page. Once the page is refreshed or
+   exited, the key will not be available to copy.
+
+   If the key cannot be copied or is lost, return to the :guilabel:`API key` page and start over,
+   creating a new API key.
+
+Webhook
+~~~~~~~
+
+A webhook is an automated notification system that instantly updates Odoo when Gelato processes,
+ships, or delivers an order, ensuring real-time tracking and minimal manual intervention.
+
+To create a webhook, go to :menuselection:`Developer --> Webhooks` under the :guilabel:`Developer`
+drop-down menu in the left menu bar. In the new page, click :guilabel:`Add Webhook` to open a
+:guilabel:`Create Webhook` form.
+
+The webhook form requires several specific configurations:
+
+- :guilabel:`URL`: This tells Gelato where to send the order updates in Odoo. Copy and paste the
+  Odoo database URL with the additional suffix `/gelato/webhook`.
+
+  .. example::
+   `https://stealthywood.odoo.com/odoo/gelato/webhook`
+
+- :guilabel:`Events`: Click into the field and select :guilabel:`order_status_updated`. Selecting
+  :guilabel:`order_status_updated` ensures Odoo receives order changes automatically.
+- :guilabel:`Method`: Click into the field and select the :guilabel:`HTTP Post` option, as this is
+   the request method used to send data from Gelato to Odoo.
```

Comment:
This rendered strangely in the HTML 🤔 
I can't pinpoint the reason, though...the syntax seems to be fine?

![Screenshot 2025-02-14 at 11 22 21 AM](https://github.com/user-attachments/assets/4a4b742f-b6b5-451b-86d9-9834d153ef1a)

Reply:
This is rendering as a [definition list](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#definition-lists) due to the one-space indent on the second line :)
```suggestion
- :guilabel:`Method`: Click into the field and select the :guilabel:`HTTP Post` option, as this is
  the request method used to send data from Gelato to Odoo.
```